available_images.md

```diff
@@ -97,7 +97,7 @@ You can pin your version by adding the version tag to your URL as follows:
 |TensorFlow 1.15.5  |inference	|No				|GPU 		| 3.6 (py36)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:1.15.5-gpu-py36-cu100-ubuntu18.04|
```

Comment:
Could you move this to be next to the CPU image for which you fixed the entry, below?

Having them both together would allow us to catch these human errors more quickly.